core/root/cluster.go

```diff
@@ -309,9 +309,9 @@ func (c clusterImpl) Update() (string, error) {
 
 	if c.controlPlane.CloudWatchLogging.Enabled && c.controlPlane.CloudWatchLogging.LocalStreaming.Enabled {
 		// Return Journald logs in a separate GoRoutine
-		quit := make(chan bool)
-		defer func() { quit <- true }()
-		go streamJournaldLogs(c, quit)
+		q := make(chan struct{})
```

Comment:
Just for my education but why did you migrate from `chan bool` to `chan struct{}`? Were there any pros/cons?